cypress/integration/common/steps.js

```diff
@@ -70,26 +88,40 @@ Then('I can see my name {string} in the dropdown menu', () => {
 
 Then('I see the login screen again', () => {
   cy.location('pathname').should('contain', '/login')
-  cy.contains(
-    'Wenn Du ein Konto bei Human Connection hast, melde Dich bitte hier an.'
-  )
+  cy.contains('If you already have a human-connection account, login here.')
 })
 
 Then('I am still logged in', () => {
   cy.get('.avatar-menu').click()
   cy.get('.avatar-menu-popover').contains(username)
 })
 
+When('I select {string} in the language menu', name => {
+  cy.get('.login-locale-switch a')
+    .click()
+  const code = find(locales, ['name', name]).code
+  cy.get(`.locale-menu-popover a.${code}`)
+    .click()
+    .wait(500)
```

Comment:
Never sleep in a browser test. Wait for something visual to appear.

See: https://gist.github.com/metaskills/1172519

Reply:
It’s caused by the cookie check. But flippig should work.